frontend/model/contracts/shared/time.js

```diff
@@ -224,7 +224,8 @@ export function timeSince (datems: number, dateNow: number = Date.now()): string
 
   if (interval >= DAYS_MILLIS * 2) {
     // Make sure to replace any ordinary space character by a non-breaking one.
-    return humanDate(datems).replace(/\x32/g, '\xa0')
+    // TODO: use .replaceAll when migrating to TS.
+    return humanDate(datems).replace(/\x20/g, '\xa0')
```

Comment:
We can't we use `.replaceAll` now? 🤔

Reply:
Just because our Flow version would complain about it being a non-existent string property. Or maybe we could add a `$FlowFixMe` comment to use it now anyway